### PR TITLE
Ons code coverage

### PR DESCRIPTION
**Pull request by frankdiox**

### CODE REVIEW

File: core/lib/ons.spec.es6 [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      let promise = ons._createAlertDialogOriginal('page.html');
+      return expect(promise).to.eventually.be.instanceof(OnsAlertDialogElement);
+    });
+  });
+
+  describe('#resolveLoadingPlaceholder()', () => {
+    it('resolves the placeholder', () => {
+      let e = document.createElement("div");
+      e.setAttribute('ons-loading-placeholder', 'page.html');
+      document.body.appendChild(e);
```
Comment (argelius) [inline core/lib/ons.spec.es6]: This element is added but never removed.

File: core/lib/ons.spec.es6 [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      ons.resolveLoadingPlaceholder();
+      expect(e.getAttribute('ons-loading-placeholder')).to.equal('undefined');
+    });
+  });
+
+  describe('#_setupLoadingPlaceHolders()', () => {
+    it('resolves the placeholder', () => {
+      let e = document.createElement("div");
+      e.setAttribute('ons-loading-placeholder', 'page.html');
+      document.body.appendChild(e);
```
Comment (argelius) [inline core/lib/ons.spec.es6]: This element too.

File: core/lib/ons.spec.es6 [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    it('enables animations', () => {
+      expect(ons._config.animationsDisabled).to.be.true;
+      ons.enableAnimations();
+      expect(ons._config.animationsDisabled).to.be.false;
+      ons.disableAnimations();
+      expect(ons._config.animationsDisabled).to.be.true;
+    });
+  });
+
+  describe('#createPopover()', () => {
```
Comment (argelius) [inline core/lib/ons.spec.es6]: Can you clean up the popover after these tests?

File: core/lib/ons.spec.es6 [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      ons.createPopover('page.html', options);
+    });
+
+    it('returns a valid popover element', () => {
+      let promise = ons.createPopover('page.html');
+      return expect(promise).to.eventually.be.instanceof(OnsPopoverElement);
+    });
+  });
+
+  describe('#_createDialogOriginal()', () => {
```
Comment (argelius) [inline core/lib/ons.spec.es6]: `#createDialog()`

File: core/lib/ons.spec.es6 [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+      ons.createDialog('page.html', options);
+    });
+
+    it('returns a valid dialog element', () => {
+      let promise = ons.createDialog('page.html');
+      return expect(promise).to.eventually.be.instanceof(OnsDialogElement);
+    });
+  });
+
+  describe('#_createAlertDialogOriginal()', () => {
```
Comment (argelius) [inline core/lib/ons.spec.es6]: `#createAlertDialog`

**Comment by frankdiox** [comment]
@argelius I think it's fixed now.

**Comment by argelius** [comment]
Thanks @frankdiox !

**Pull request merged**